deployability/modules/allocation/static/specs/os.yml

```diff
@@ -367,24 +367,24 @@ aws:
   windows-desktop-10-amd64:
     ami: ami-0a747df120215911a
     zone: us-east-1
-    user: Administrator
-  windows-desktop-11-amd64:
-    ami: ami-09d8cef159442d5b0
-    zone: us-east-1
-    user: Jenkins
+    user: wazuh-user
+  #windows-desktop-11-amd64: Issue related https://github.com/wazuh/wazuh-automation/issues/1582
+  #  ami: ami-0115c7e59b426bffb
+  #  zone: us-east-1
+  #  user: Jenkins
```

Comment:
Remove it and create an issue with DTT2 objective please

Reply:
Issue created: https://github.com/wazuh/wazuh-automation/issues/1582

Reply:
Done: 1ffd41c6b